src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,6 +68,20 @@ public RobotContainer() {
 
   public void disabledInit() {
     subsystems.disable();
+    coastModeTimer.restart();
+    subsystems.algaeArm.disable();
+    subsystems.algaeGrabber.disable();
+    subsystems.climber.disable();
+    subsystems.coralArm.disable();
+    subsystems.drivetrain.disableAutoOrientation();
+    subsystems.elevator.disable();
```

Comment:
Disabling the subsystems is now handled by the call to `subsystems.disable()`. It is not necessary to do so explicitly here.

We're not using auto-orientation this year and should probably just remove the code.

Reply:
> We're not using auto-orientation this year and should probably just remove it from the code.

An alternative would be to implement the `ActiveSubsystem` interface on the `Swerve` subsystem and have its `disable()` method call `disableAutoOrientation()`.